index.html

```diff
@@ -4,7 +4,121 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Surbhi Kukreti" />
+    <meta
+      name="description"
+      content="checkBox Home page a quiz app on web developement"
+    />
+    <link rel="stylesheet" href="/cssBundle/checkBox.css" />
     <title>checkBox</title>
   </head>
-  <body></body>
+  <body>
+    <!-- header -->
+    <header class="nav-head">
+      <!-- logo -->
+      <div class="flex-row flex-center gap-sm">
+        <img src="/media/icon/box.svg" alt="Profile icon" class="logo-icon" />
+
+        <a href="/index.html" class="text-dec-none">
+          <div class="flex-row">
+            <h1 class="marg-un logo">check</h1>
+            <h1 class="marg-un logo1">Box</h1>
+          </div>
+        </a>
+      </div>
+      <!-- end of logo -->
+
+      <ul class="nav-links">
+        <li>
+          <a class="nav-link" href="#"> Categories</a>
+        </li>
+        <li>
+          <a class="nav-link" href="#"> High-Scores </a>
+        </li>
+
+        <!-- Profile  -->
+        <li>
+          <a class="nav-link outline-btn" href="#">
+            <img
+              src="/media/icon/user.svg"
+              alt="Profile icon"
+              class="nav-icon"
+            />
+          </a>
+        </li>
+      </ul>
+    </header>
+    <!-- end of header -->
+
+    <!-- start of main -->
+    <main class="main-cont flex-center">
+      <div class="flex-col flex-center width80">
+        <h3 class="mg-sm">Welcome to checkBox,</h3>
+        <h3 class="marg-un">
+          A quiz app specifically designed for developers to have fun and learn
+          altogether.
+        </h3>
+
+        <h2>Are you ready?</h2>
+        <h1>Then lets begin!</h1>
```

Comment:
The hierarchy of heading is semantically incorrect here.  Is there a specific reason for writing this. let me know your thoughts about the same.

Reply:
i did that to grab attention where i intended to...i'll see if i can use font size.

Reply:
Yes, you can use font-size and font-weight property to achieve this.